ad/movement/dacl/grant-rights.md

```diff
@@ -16,12 +16,18 @@ If attacker can write an ACE (`WriteDacl`) for a container or organisational uni
 Impacket's dacledit (Python) can be used with the `-inheritance` flag for that purpose ([PR#1291](https://github.com/fortra/impacket/pull/1291)).
 {% endhint %}
 
+{% hint style="info" %}
+**`adminCount=1`**
+
+If `GenericAll`, `GenericWrite` or `Manage Group Policy Links` privileges are available against an Organisational Unit (OU), then it's possible to compromise its child users and computers with `adminCount=1`.
```

Comment:
I'd add
- `In April 2024, [Synkactiv explained](ARTICLE) that if [...]` at the beginning
- `through "gPLink spoofing". This can be performed with [OUned.py](TOOL)` at the end.

Reply:
I would also move this content into the "ACE inheritance" part above, and also in the same section inside the following page: https://www.thehacker.recipes/ad/movement/dacl

Reply:
Done in commit: GITBOOK-4.